docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md

```diff
@@ -107,7 +107,7 @@ The `Config` object, together with `SlickConfig` and `DefaultSlickConfig`, were
 
 ## `SlickPlayIteratees` was removed
 
-If you were using `SlickPlayIteratees.enumerateSlickQuery` to stream data from the database, you will be happy to know that doing so became a lot easier. Slick 3 implements the [reactive-streams] SPI, and Play 2.4 provides a utility class to handily convert a reactive stream into a Play enumerator.
+If you were using `SlickPlayIteratees.enumerateSlickQuery` to stream data from the database, you will be happy to know that doing so became a lot easier. Slick 3 implements the [reactive-streams] specification, and Play 2.4 provides a utility class to handily convert a reactive stream into a Play enumerator.
```

Comment:
Maybe you can replace it by:

```markdown
... the [reactiv-streams] [SPI] (Service Provider Interface) ... 
```

Adding the following link at:

http://en.wikipedia.org/wiki/Service_provider_interface